src/components/common/signupform/signupformcontainer.js

```diff
@@ -0,0 +1,32 @@
+import Link from 'next/link'
+import styles from './signupcontainer.module.css'
+import CustomizableAvatar from '../CustomizableAvatar'
+import SignupForm from './SignupForm'
+import images from '../../../images/index'
+
+function SignupFormContainer() {
+  return (
+    <main>
+      <div className={styles.div}>
+        <CustomizableAvatar
+          height={100}
+          width={100}
+          src={images.signupAvatar}
+          alt='main avatar'
+        />
+      </div>
+      <h2 className={styles.header}>Create an account</h2>
+      <h4 className={styles.sub}>
+        Lets get started with creating your community!
+      </h4>
+      <SignupForm />
```

Comment:
I believe our wireframes indicated an `<Aside>` component to be side-by-side with the SignUp form.

That component will be available with @Sheyitofunmi's [PR](https://github.com/AritDeveloperCircle/findyourpeople/pull/46)

Please leave a comment here to indicate that intention. 👍🏾

Reply:
> I believe our wireframes indicated an `<Aside>` component to be side-by-side with the SignUp form.
> 
> That component will be available with @Sheyitofunmi's [PR](https://github.com/AritDeveloperCircle/findyourpeople/pull/46)
> 
> Please leave a comment here to indicate that intention. 👍🏾

Yes, I'm just waiting for her PR to come in to complete the full signup page